prow/config/config.go

```diff
@@ -381,6 +381,12 @@ type Plank struct {
 	// DefaultDecorationConfig are defaults for shared fields for ProwJobs
 	// that request to have their PodSpecs decorated
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
+
+	// DecorationConfigByRepo holds the default decoration config for specific repositories.
+	// This config will be used only on Presubmits and Postsubmits and the missing fields will
```

Comment:
To be similar to the other options, we should:

```
// Use `org/repo`, `org` or `*` as a key
```

Reply:
addressed